features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -745,6 +745,28 @@ CREATE TRIGGER IDP_ROLE_TRIG NO CASCADE
                     SET (NEW.ID) = (NEXTVAL FOR IDP_ROLE_SEQ);
                 END
 /
+CREATE TABLE IDP_GROUP (
+			ID INTEGER NOT NULL,
+			IDP_ID INTEGER NOT NULL,
+			TENANT_ID INTEGER NOT NULL,
+			GROUP_NAME VARCHAR(254) NOT NULL,
```

Comment:
Any reason to define the size as VARCHAR(254) instead of VARCHAR(255)?

Reply:
Followed the IDP_ROLE table approach here.
However its not necessary based on https://stackoverflow.com/questions/10043556/mysql-why-varchar254-and-not-varchar255
Changed the size to 255 in 263284864bfaef496ae5df88973bc8cddb70f749